### ISSUE TITLE
CLI: bad error message when missing required field not provided

### ISSUE THREAD

**Issue by dustymabe** [fedora-infra/bodhi#2463]
`bodhi-client-2.12.2-3.fc27.noarch`

I tried to create an update thinking it would pop open a window and let me fill in missing info:

```
$ bodhi updates new ignition-0.26.0-0.2.git7610725.el7                                                                                                               
Password:
Required
You may not specify an empty list of builds.
ACL validation mechanism was unable to determine ACLs.
```

So it compains I didn't specify a build, but I did. OK reading through the man page I see there is a --type field. Let me try adding that:

```
$ bodhi updates new --type newpackage ignition-0.26.0-0.2.git7610725.el7                                                                                             
Required
You may not specify an empty list of builds.
ACL validation mechanism was unable to determine ACLs.
```

nope.. what about notes??

```
$ bodhi updates new --type newpackage --notes "new package" ignition-0.26.0-0.2.git7610725.el7 
================================================================================
     ignition-0.26.0-0.2.git7610725.el7
================================================================================
   Update ID: FEDORA-EPEL-2018-7797712575
Content Type: rpm
     Release: Fedora EPEL 7
      Status: pending
        Type: newpackage
       Karma: 0
   Autokarma: False  [-3, 3]
     Request: testing
       Notes: new package
   Submitter: dustymabe
   Submitted: 2018-06-27 14:40:48
    Comments: bodhi - 2018-06-27 14:40:48 (karma 0)
              This update has been submitted for testing by
              dustymabe.

  https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2018-7797712575
```

bingo. It would be useful if the error message marked those two as required arguments ?

**Comment by ghost** [comment]
Might be related to #2214

**Comment by dustymabe** [comment]
indeed. @bowlofeggs feel free to dup this

**Comment by cverna** [comment]
Closed as duplicate of #2214
